cmd/wnode-status/config.go

```diff
@@ -106,5 +108,10 @@ func makeNodeConfig() (*params.NodeConfig, error) {
 	nodeConfig.IPCEnabled = *ipcEnabled
 	nodeConfig.RPCEnabled = *httpEnabled
 
+	if whisperConfig.Standalone {
```

Comment:
Do we need `whisperConfig.Standalone`? We could just check for `*standalone` and set `nodeConfig.BootClusterConfig.Enabled` and `nodeConfig.BootClusterConfig.BootNodes`.

Reply:
Single responsibility... but i'll do what you asked.

Reply:
Yeah, that what I meant :) Your code suggested that `whisperConfig` was responsible if node should you use bootnodes and it's `nodeConfig.BootClusterConfig.*` responsibility.